apps/plate-editor/src/app/globals.css

```
@@ -0,0 +1,477 @@
+  --color-selection-foreground: var(--selection-foreground);
+
+  --color-brand: var(--brand);
+  --color-highlight: var(--highlight);
+
+  --font-heading:
+    "var(--font-heading)", "ui-sans-serif", "-apple-system",
+    "BlinkMacSystemFont", "Segoe UI Variable Display", "Segoe UI", "Helvetica",
+    "Apple Color Emoji", "Arial", "sans-serif", "Segoe UI Emoji",
+    "Segoe UI Symbol", "Noto Color Emoji";
+  --font-mono:
+    "var(--font-mono)", ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas,
```

Stylelint (17.3.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)